libqtile/manager.py

```diff
@@ -1196,6 +1196,9 @@ def cmd_critical(self):
         logger.setLevel(logging.CRITICAL)
         logger.critical('Switching to CRITICAL threshold')
 
+    def cmd_loglevel(self):
+        return (logging.getLevelName(logger.level), logger.level)
```

Comment:
Why do we need both? Can't we just have enough with one?

Reply:
The logger.level is represented by numerical values. [a link of the reference](https://docs.python.org/3/howto/logging.html#logging-levels).
It is difficult to understand for me, so I wrote both. Is it better a idea to remove the left one (loggin.getLevelName(logger.level))?

Reply:
What would you think of doing two commands, `cmd_loglevel` and `cmd_loglevelname`?

Reply:
I think it is good. I'll make a commit.

Reply:
Sorry to bother you again. I changed the test.